acme_diags/parameter/zonal_mean_2d_parameter.py

```diff
@@ -1,11 +1,35 @@
 import numpy
 from .core_parameter import CoreParameter
-
+from acme_diags.driver.utils.general import strictly_increasing,strictly_decreasing,non_increasing,non_decreasing,monotonic
```

Comment:
Same comments as last file. Actually, these files are so similar that you might consider creating a `Mean2dParameter` class for both to inherit from.

Reply:
Perhaps that could be a separate GitHub issue; since these files existed before this pull request, it wouldn't make sense to do that refactor in this pull request.

Reply:
True. Let's keep it this way for now.